diff --git a/tests/test_parser.py b/tests/test_parser.py
index 7cd7dd8..6abc253 100644
--- a/tests/test_parser.py
+++ b/tests/test_parser.py
@@ -273,7 +273,7 @@ def _make_full_earley_test(LEXER):
             grammar = """
             ANY:  /[a-zA-Z0-9 ]+/
             a.2: "A" b+
-            b.2: "B"
+            b.2: "B" 
             c:   ANY
 
             start: (a|c)*
@@ -1081,6 +1081,20 @@ def _make_parser_test(LEXER, PARSER):
                 'y'])
 
 
+        def test_relative_import(self):
+            grammar = """
+            start: NUMBER WORD
+
+            %import .grammars.test.NUMBER
+            %import common.WORD
+            %import common.WS
+            %ignore WS
+
+            """
+            l = _Lark(grammar)
+            x = l.parse('12 lions')
+            self.assertEqual(x.children, ['12', 'lions'])
+
         def test_multi_import(self):
             grammar = """
             start: NUMBER WORD
@@ -1093,6 +1107,17 @@ def _make_parser_test(LEXER, PARSER):
             x = l.parse('12 toucans')
             self.assertEqual(x.children, ['12', 'toucans'])
 
+        def test_relative_multi_import(self):
+            grammar = """
+           start: NUMBER WORD
+
+           %import .grammars.test (NUMBER, WORD, WS)
+           %ignore WS
+
+           """
+            l = _Lark(grammar)
+            x = l.parse('12 capybaras')
+            self.assertEqual(x.children, ['12', 'capybaras'])
 
         def test_relative_multi_import(self):
             l = _Lark_open("test_relative_multi_import.lark", rel_to=__file__)
