diff --git a/tests/test_parser.py b/tests/test_parser.py
index 19b09f2..4f1f1e6 100644
--- a/tests/test_parser.py
+++ b/tests/test_parser.py
@@ -191,29 +191,6 @@ def _make_full_earley_test(LEXER):
 
 
         @unittest.skipIf(LEXER=='dynamic', "Only relevant for the dynamic_complete parser")
-        def test_earley3(self):
-            "Tests prioritization and disambiguation for pseudo-terminals (there should be only one result)"
-
-            grammar = """
-            start: A A
-            A: "a"+
-            """
-
-            l = Lark(grammar, parser='earley', lexer=LEXER)
-            res = l.parse("aaa")
-            self.assertEqual(res.children, ['a', 'aa'])
-
-        def test_earley4(self):
-            grammar = """
-            start: A A?
-            A: "a"+
-            """
-
-            l = Lark(grammar, parser='earley', lexer=LEXER)
-            res = l.parse("aaa")
-#            print(res.pretty())
-            self.assertEqual(res.children, ['aaa'])
-
         def test_earley_repeating_empty(self):
             # This was a sneaky bug!

